### PR TITLE
SPV_EXT_shader_image_int64 clarify Int64Atomics interaction

### PR DESCRIPTION
**Pull request by sfricke-samsung**

### CODE REVIEW

**Review by Tobski — APPROVED** [review]

File: extensions/EXT/SPV_EXT_shader_image_int64.asciidoc [hunk 1]
```diff
@@ -94,6 +94,7 @@ Modify Section 3.31, "Capability", adding these rows to the Capability table:
 2+^| Capability ^| Enabling Capabilities
 | 5016 | *Int64ImageEXT* +
 Uses atomic, load, or store instructions on images with 64-bit integer types.
+If using atomics, *Int64Atomics* must be enabled.
```
Comment (sfricke-samsung) [inline extensions/EXT/SPV_EXT_shader_image_int64.asciidoc:97]:
```suggestion
If using atomics, *Int64Atomics* must be declared.
```

File: extensions/EXT/SPV_EXT_shader_image_int64.html [hunk 2]
```diff
@@ -901,7 +902,8 @@ <h3 id="_3_31_capabilities">3.31 Capabilities</h3>
 <tr>
 <td align="left" valign="top"><p class="table">5016</p></td>
 <td align="left" valign="top"><p class="table"><strong>Int64ImageEXT</strong><br />
-Uses atomic, load, or store instructions on images with 64-bit integer types.</p></td>
+Uses atomic, load, or store instructions on images with 64-bit integer types.
+If using atomics, *Int64Atomics* must be enabled.</p></td>
```
Comment (sfricke-samsung) [inline extensions/EXT/SPV_EXT_shader_image_int64.html:906]:
```suggestion
If using atomics, *Int64Atomics* must be declared.</p></td>
```

**Review by sfricke-samsung — COMMENTED** [review]
@johnkslang This good? Simple improvement

**Review by Tobski — APPROVED** [review]
Sorry didn't realise this was waiting on me - LGTM!

**Pull request merged**